pkg/app/piped/cloudprovider/kubernetes/helm.go

```diff
@@ -99,7 +108,7 @@ type helmRemoteGitChart struct {
 }
 
 func (c *Helm) TemplateRemoteGitChart(ctx context.Context, appName, appDir, namespace string, chart helmRemoteGitChart, gitClient gitClient, opts *config.InputHelmOptions) (string, error) {
-	// Firstly, we need to download the remote repositoy.
+	// Firstly, we need to download the remote repository.
```

Comment:
This has no relationship with the purpose of the PR, but it is a simple typo, so I fixed it.